terraform/environments/delius-core/application_variables.json

```diff
@@ -22,7 +22,9 @@
       "rds_enabled_cloudwatch_logs_exports": ["alert", "audit"],
       "rds_performance_insights_enabled": "false",
       "rds_skip_final_snapshot": "true",
-      "rds_apply_immediately": "true"
+      "rds_apply_immediately": "true",
+      "backup_schedule": "cron(0 19 * * ? *)",
```

Comment:
I think it may be worth specifying what this backup schedule is for in variable name.